test/e2e-tests-yaml.sh

```diff
@@ -0,0 +1,21 @@
+source $(dirname $0)/e2e-common.sh
+
+set -o errexit
+set -o pipefail
+
+for op in apply delete;do
+    # Apply TriggerTemplates
+    for file in $(find ${REPO_ROOT_DIR}/examples/triggertemplates/ -name *.yaml | sort); do
+    kubectl ${op} -f ${file}
+    done
+
+    # Apply TriggerBindings
+    for file in $(find ${REPO_ROOT_DIR}/examples/triggerbindings/ -name *.yaml | sort); do
+    kubectl ${op} -f ${file}
+    done
+
+    # Apply EventListeners
+    for file in $(find ${REPO_ROOT_DIR}/examples/eventlisteners/ -name *.yaml | sort); do
+    kubectl ${op} -f ${file}
+    done
+done
```

Comment:
I don't think that this test actually tracks the failures. If one resource fails to be created, then the script will still return a success. We can probably track a return code like elsewhere in this PR:
```
failed=0
...
kubectl ${op} -f ${file} || failed=1
...
(( failed )) && fail_test
success
```